app/controllers/application_controller.rb

```diff
@@ -3,12 +3,6 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
-  helper_method :current_division
-
-  def current_division
```

Comment:
Ah.  Thanks!  I hadn't realized that.  I'm curious what will drive the selection here.  Will each division have their own public site?  It's not clear to me if this should be driven by the same logic and or helper method or made distinct.  Thoughts?